skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillDTO.java

```diff
@@ -0,0 +1,42 @@
+package com.RDS.skilltree.Skill;
+
+import com.RDS.skilltree.User.UserDTO;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Getter
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SkillDTO {
+    private UUID id;
+    private SkillType type;
+    private String name;
+    private Set<UserDTO> users;
+
+    public static SkillDTO toDto(SkillModel skillModel) {
+        return SkillDTO.builder()
+                .id(skillModel.getId())
+                .name(skillModel.getName())
+                .type(skillModel.getType())
+                .build();
+    }
```

Comment:
Should we remove this?

Reply:
why? just curious @Ajeyakrishna-k this is without the users being sent with it.

Reply:
- [ ] Try looking into libraries for conversion